Gemfile

```diff
@@ -82,6 +82,7 @@ end
 
 group :test do
   gem 'faker'
+  gem 'faker-medical'
```

Comment:
Where it's used in specs the expectation is checking the same generated data or that the generated data was transformed correctly (e.g. ssn minus dashes)